Data/ByteString/Internal.hs

```diff
@@ -65,6 +65,7 @@ module Data.ByteString.Internal (
 
         -- * Standard C Functions
         c_strlen,
+        c_strstr,
```

Comment:
It's not clear to me this should be exported here.

Reply:
Sure. Moving the foreign import to where it is used...